integration/test/Test/FeatureFlags.hs

```diff
@@ -181,3 +182,53 @@ checkFeature feature user tid expected = do
   bindResponse (Public.getFeatureConfigs user) $ \resp -> do
     resp.status `shouldMatchInt` 200
     resp.json %. feature `shouldMatch` expected
+
```

Comment:
Could you add a comment expanding a bit on what we are testing, what the expected behaviours are?